client/packages/host/package.json

```diff
@@ -23,7 +23,7 @@
     "start-remote": "webpack-cli serve --env API_HOST='https://demo-open.msupply.org'",
     "start-local": "webpack-cli serve --env API_HOST='http://localhost:8000'",
     "build": "webpack --env production",
-    "build-dev": "webpack",
+    "build-stats": "webpack --env stats",
```

Comment:
should it also have production env ?

Reply:
good catch! yes 🙏  the bundle size will be bigger when running a debug build!!
maybe, just maybe, 16MB instead of 2 or 3.. 🤔